### ISSUE TITLE
Contenue média

### ISSUE THREAD

**Issue by Tchepga** [mongulu-cm/icons-fasaha#16]
Regarder s'il est possible d'avoir du contenue média un peu propre à l’Afrique noir à partir de dalle-2

**Comment by Tchepga** [comment]
- chef village (prompt : icon chief of bamileke's village, digital painting, vector image)
![chef 1](https://user-images.githubusercontent.com/34720602/220769120-7da32e07-87ca-4fb4-b229-ab171d30c7cc.png)
![chef 2](https://user-images.githubusercontent.com/34720602/220769115-297f7a4a-89e4-4a6d-904a-f9b47eb2c2f4.png)

**Comment by Tchepga** [comment]
icon chief of bamileke's village, flaticon style, fontawesome style
![chef3](https://user-images.githubusercontent.com/34720602/220770485-310002ff-2a5c-4a81-adfd-627c9131e7e3.png)
![chef4](https://user-images.githubusercontent.com/34720602/220770491-fbd28d77-6991-4a05-8884-9ba261b823f6.png)

**Comment by Tchepga** [comment]
@billmetangmo  on peut s'expirer des librairies d'icone come fontawasone, material icons

**Comment by billmetangmo** [comment]
Oh c'est très cool ces images @Tchepga .ça génère les images en SVG ?
Very good job, je pense qu'on tient un bon bout là. 

- [x] Tu peux essayer de voir peut-être juste que si c'est bamiléké on essaie de trouver un moyen avec DALLE qu'il puisse utiliser des élements spécifiques comme ceux du ndop par exemple ? Sinon peut-on ( **ce sera pour plutard une v2**) tuner DALLE à partir de plusisuers images. Je crois que c'est possible avec StableDiffusion ( cf https://civitai.com/models/6526/studio-ghibli-style-lora )
- [ ] Aussi, il faut que tu voies sur quel plateforme on pourrait uploader ces icones avec le prompt comme message : existante comme lexica.art ou autres ou auto-hébergement avec leurs avantages/incovénients.

**Comment by billmetangmo** [comment]
Une images intéressante vur sur Lexica: https://lexica.art/prompt/9f9d9624-2e81-46d9-b9fd-ea353863f899

**Comment by Tchepga** [comment]
OK @billmetangmo je vais regarder. Il est possible de faire des icones mais il génére des png. Il faudrait après jusque convertir en svg. On pourrait hebergé une petite app sur aws ou oracle. Pour faire les interfaces pour gérer ca, c'est assez facile (8h max) de développement.

**Comment by billmetangmo** [comment]
Je pense qu'il est toujours préférable de partir sur une solution existante que de redévelopper. 
J'ai vu qu'il y a ça https://github.com/tomayac/SVGcode pour transformer les JPG, PNG, GIF, WebP, AVIF, etc. en vector graphics in SVG format.Tu peux me dire si ça répond au besoin ? Si oui, on pourra juste l'auto-héberger.

**Comment by billmetangmo** [comment]
Une autre possibilité c'est que dans l'évelaution des plateformes possibles, tu rajoutes le critère de savoir si celle-ci permet soit de transformer directement l'image en diférrents formats dont SVG ou s'il existe des webhooks pouvant être déclenchés quand on uploade une image ( soit car supporté nativement ou si celle-ci stocke les images dans storage comme S3).

**Comment by Tchepga** [comment]
https://github.com/tomayac/SVGcode ca résout partiellement le problème. Sur certaines images, ils remplit avec une couleur (black pour le cas d'espace). La solution serait de modifier manuelement le fichier svg ou de générer des justes les contours des images sur dalle2

**Comment by billmetangmo** [comment]
Je comprends pas. Tu veux dire qu'il remplit le vide avec une couleur ? Tu peux envoyer une pciture ( Avant | Après) de ce que tu dis que je le visualise.

**Comment by Tchepga** [comment]
@billmetangmo oui il remplit le vide par le noir sur ceux ce j'ai essayé. Si je me rappelle bien c'est la propriété fill de la balise svg qui est mis en noir <svg fill="#000"

**Comment by Tchepga** [comment]
Je vais refaire un exemple et j'envoie

**Comment by billmetangmo** [comment]
Yes @Tchepga .Fais une capture d'écran . Tu peux utiliser  https://www.loom.com/ ou bien https://obsproject.com/fr

**Comment by Tchepga** [comment]
https://www.loom.com/share/98d434ee059847c09b7a380a6d8b8a7a

**Comment by Tchepga** [comment]
https://www.loom.com/share/2954ae87b73c4ff3b4378510076bf9eb

**Comment by Tchepga** [comment]
https://www.loom.com/share/d68c6fbc4f30474f9a1694b074e995b4
@billmetangmo  ca ne marche qu'avec chrome, je crois bien

**Comment by billmetangmo** [comment]
Ah je vois. Je comprends mieux

**Comment by billmetangmo** [comment]
Tu peux ouvrir une issue sur le projet https://github.com/tomayac/SVGcode avec ce soucis. Ne sait-on jamais ?
Je vais creuser d'autres outils alors de mon côté. Tu as pu faire un test avec DALLE et d'autres paramètres ?

**Comment by billmetangmo** [comment]
@Tchepga il faudrait faire un test aussi avec https://www.bing.com/images/create & https://www.adobe.com/sensei/generative-ai/firefly.html

**Comment by Tchepga** [comment]
en utilisant firefly. Je fais aussi quelque test sur photoshop directement
![Firefly Calebasse Afrique - 26388](https://github.com/mongulu-cm/icons-fasaha/assets/34720602/abafba33-f123-4774-9b74-960abe930199)

**Comment by Tchepga** [comment]
![Firefly masque malien 26388](https://github.com/mongulu-cm/icons-fasaha/assets/34720602/d8856e65-01d6-4be1-8c22-c29f3b00b36f)
![Firefly Calebasse Afrique - 26388](https://github.com/mongulu-cm/icons-fasaha/assets/34720602/f39426a7-307e-43d7-8f03-c1e902723f3c)

**Comment by Tchepga** [comment]
chef village (prompt : icon chief of bamileke's village, digital painting, vector image)
![Firefly icon chief of bamileke's village, digital painting, vector image 8088](https://github.com/mongulu-cm/icons-fasaha/assets/34720602/fc0a8b21-beb2-4a69-ad5a-9939fde6b416)

**Comment by billmetangmo** [comment]
Cool. C'est vraiment plus beau que dalle et plus dans le thème que Midjourney

**Comment by Tchepga** [comment]
Firefly signe chacune de ces images et malheureusement je n'ai pas encore toujours comment générer directement les svg. Il faudrait un prompt pour générer une image sans arrière plan. 
Pour le moment toute les images téléchargés sont signés.

**Comment by Tchepga** [comment]
En utilisant lexica.art
![icon chief of bamileke's village](https://github.com/mongulu-cm/icons-fasaha/assets/34720602/0c1dd334-8fae-4185-b710-0cb069dd870c)
![icon chief of bamileke's village (1)](https://github.com/mongulu-cm/icons-fasaha/assets/34720602/5d1c20c1-4d37-45d2-b812-f44849519cd2)

**Comment by billmetangmo** [comment]
Cool. Avec toutes ses expérimentations d qui donne des images intéressantes mais pas super proche du contexte camerounais.

Je pense qu une solution serait d utiliser les images de masques ( j en avais partage dans une issue) puis utiliser des modèles:

- [x] - qui permettent de partir d une image et de la modifier ( il y a mid journey qui fait ça. Qui d autre ?)
- [x] - ou de créé un prompt à partir d' une image distante et générer une image à partir de ce prompt. Cf https://korben.info/comment-plagier-photo.html

**Comment by billmetangmo** [comment]
![4EA07C96-C039-4B2B-883B-6FE3828CBF7B](https://github.com/mongulu-cm/icons-fasaha/assets/25366207/45ad5665-82c2-4ce3-86f0-1c2a140a4080)

**Comment by billmetangmo** [comment]
Les résultats n'avaient pas été concluant de ce côté là, j'abandonne cette partie @Tchepga

**Comment by billmetangmo** [comment]
Quoique on peut visiblement utiliser ControlNet + 1 Canny avec Stable diffusion pour controler le modèle: https://youtu.be/icrdVc0l2a0?t=2505

**Comment by billmetangmo** [comment]
Encore plus simple avec ComfyUI, visiblement il faut passer par https://github.com/comfyanonymous/ComfyUI
Chaine youtube intéressante: https://www.youtube.com/watch?v=8sTBk6ZhPi0
